application_sandbox/sample_application_framework/sample_application.h

```diff
@@ -268,6 +273,10 @@ class Sample {
       LOG_ASSERT(!=, data_->logger(), VK_FORMAT_UNDEFINED,
                  depth_stencil_format_);
     }
+    if (options_.use_high_precision_depth && options_.enable_depth_buffer) {
```

Comment:
I think this may need to be changed to an "else if". If someone uses both enable_stencil and use_high_precision_depth, then this will clobber the stencil format selection. The GetSupportedDepthStencilFormat function will attempt to use a high precision depth format so I don't think we need to enumerate all the options between enable_stencil and use_high_precision_depth.

Reply:
Will do that